### DISCUSSION TITLE
GIF in vercel

### DISCUSSION THREAD

**Discussion by ghost** [neuefische/questions#531] [Web] [answered]
Locally in my public folder I have two GIF's.
I implemented them via Image component .

Everything works fine in the localhost, but in vercel they fail to load!
Why does it happen and what can I do to fix it?

**Comment by doemser** [comment, marked as answer]
Can you try if this approach fits your needs?

https://github.com/vercel/next.js/discussions/38823

**Reply by ghost** [reply]
Thanks .. works perfect, when i am not using src="/public/etc"
